Makefile

```diff
@@ -21,7 +21,11 @@ darwin:
 	@ go generate
 	@ GOOS=darwin GOARCH=amd64 go build -ldflags="-X 'github.com/miniflux/miniflux/version.Version=$(VERSION)' -X 'github.com/miniflux/miniflux/version.BuildDate=$(BUILD_DATE)'" -o $(APP)-darwin-amd64 main.go
 
-build: linux linux-arm darwin
+freebsd:
+        @ go generate
+        @ GOOS=freebsd GOARCH=amd64 go build -ldflags="-X 'githug.com/miniflux/miniflux/version.Version=$(VERSION)' -X 'github.com/miniflux/miniflux/version.BuildDate=$(BUILD_DATE)'" -o $(APP)-freebsd-amd64 main.go
+
+build: linux linux-arm darwin miniflux
```

Comment:
Same here?

Reply:
yes, fixed in https://github.com/miniflux/miniflux/pull/86/commits/44da0e83c6101497b40df43b7d2840f5bc49ad55